README.md

```diff
@@ -1,3 +1,5 @@
+![CI](https://github.com/SafeBoda/throttlex/workflows/CI/badge.svg)
```

Comment:
Let's move it after the project description (after like 5 `> ### Throttle/Circuit Breaker Utilities`)